src/oedisi/types/common.py

```diff
@@ -10,6 +10,7 @@
 class DefaultFileNames(str, Enum):
     INPUT_MAPPING = "input_mapping.json"
     STATIC_INPUTS = "static_inputs.json"
+    WIRING_DIAGRAM = "system.json"
```

Comment:
What's the use of this? It seems to be completely internal to the broker?

Reply:
DefaultFileNames.WIRING_DIAGRAM is used in the sigdal example. ~\sgidal-example\broker\server.py line 209.
I want to be consistent and not use hard-coded file names

Reply:
moved it after reading comment on the example repo